alot/ui.py

```diff
@@ -317,7 +317,7 @@ def select_or_cancel(text):
 
         def cerror(e):
             logging.error(e)
-            self.notify('completion error: %s' % e.message,
+            self.notify('completion error: %s' % str(e),
```

Comment:
Won't the string interpolation cast to string anyways?

Reply:
I don't know, I never use % formatting myself. This might have also been done by 2to3